cypress/Objects/TimeLoggin.js

```diff
@@ -0,0 +1,25 @@
+class TimeLoggin{
+    validateUrl() {
+        cy.url().should('include', '/time-logging')
+        }
```

Comment:
Indent for this line is too far right, please use same approach as in the rest of the file